app/jobs/attach_files_to_file_set_job.rb

```diff
@@ -0,0 +1,75 @@
+# Converts UploadedFiles into FileSets and attaches them to works.
+class AttachFilesToFileSetJob < AttachFilesToWorkJob
```

Comment:
What change was needed here that required us overriding the (presumably) upstream `AttachFilesToFileSetJob`?